### PR TITLE
 Use new testutils document fixtures in tests

### PR DESCRIPTION
**Pull request by benvand**
Will fail until https://github.com/alphagov/digitalmarketplace-test-utils/pull/11/files is merged

### CODE REVIEW

**Comment by jonodrew** [comment]
I'm trying to run the tests on these and getting this error:
```
ImportError while importing test module '/Users/jonathankerr/digital-marketplace/digitalmarketplace-supplier-frontend/tests/app/main/test_frameworks.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/app/main/test_frameworks.py:19: in <module>
    from dmtestutils.fixtures import valid_jpeg_bytes
E   ModuleNotFoundError: No module named 'dmtestutils.fixtures'
```

I've pulled `digitalmarketplace-test-utils`; is there anything else I need to make this run right?

File: requirements-dev.txt [hunk 1]
```diff
@@ -12,5 +12,5 @@ python-coveralls==2.5.0
 requests-mock==0.6.0
 watchdog==0.8.3
 
-git+https://github.com/alphagov/digitalmarketplace-test-utils.git@1.5.0#egg=digitalmarketplace-test-utils==1.5.0
+git+https://github.com/alphagov/digitalmarketplace-test-utils.git@2.0.0#egg=digitalmarketplace-test-utils==2.0.0
```
[the remark this answers is not in the document]
Comment (benvand) [reply requirements-dev.txt]: Done

**Comment by benvand** [comment]
@jonodrew I think you need to manually run the `pip install -r requirements-dev.txt`

**Comment by benvand** [comment]
Huh, and that's why we need to get travis set up on this repo!

**Comment by benvand** [comment]
Funnily enough the above is actually fixed in my next pr 
https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/946

**Comment by benvand** [comment]
https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/946/files#diff-b67911656ef5d18c4ae36cb6741b7965R61

**Pull request merged**